cmd/zpool/zpool_main.c

```diff
@@ -1495,12 +1495,23 @@ find_spare(zpool_handle_t *zhp, void *data)
 	return (0);
 }
 
+typedef struct status_cbdata {
+	int		cb_count;
+	int		cb_name_flags;
+	int		cb_namewidth;
+	boolean_t	cb_allpools;
+	boolean_t	cb_verbose;
+	boolean_t	cb_explain;
+	boolean_t	cb_first;
+	boolean_t	cb_dedup_stats;
+} status_cbdata_t;
+
 /*
  * Print out configuration state as requested by status_callback.
  */
 static void
-print_status_config(zpool_handle_t *zhp, const char *name, nvlist_t *nv,
-    int namewidth, int depth, boolean_t isspare, int name_flags)
+print_status_config(status_cbdata_t *cb, zpool_handle_t *zhp, const char *name,
+    nvlist_t *nv, int depth, boolean_t isspare)
```

Comment:
Let's leave `zpool_handle_t *` as the first argument for consistency.  I'd suggest making it the second arg.